### PR TITLE
⚗️ add flask with stubbed-out endpoint

### PR DESCRIPTION
**Pull request by clanam**
leaving application.py as-is since it looks like it was intended to be run as a script.  will run create-react-app next to create a field for the ui folks (like me!) to play in. 🐿️

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from flask import Flask, jsonify
+import json
+import pdb
+
+# instantiate the app
+app = Flask(__name__)
+app.config.from_object(__name__)
+
+@app.route('/emoji', methods=['GET'])
```
Comment (gmkohler) [inline app.py]: this is the correct pluralization of emoji.

File: app.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+def get_emoji_list():
+    try:
+        with open('results.json') as json_file:
```
Comment (gmkohler) [inline app.py]: FYI @nifengle, if this gets merged in before yours, we'll have two seed files in the app.

File: app.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+            data = json.load(json_file)
+            return jsonify(data)
+    except:
+        return jsonify('error: could not fetch emoji :(')
```
Comment (gmkohler) [inline app.py]: Curious, does this return a 500 response still?

File: app.py (hunk 2 above)
Comment (nifengle) [reply app.py]: I'm adding the seed file to gitignore

File: app.py (hunk 1 above)
Comment (clanam) [reply app.py]: 😝

File: app.py (hunk 2 above)
Comment (clanam) [reply app.py]: don't add it to gitignore -- it's useful for devs to have it available for now. i'll point to wherever you put it @nifengle

File: app.py (hunk 3 above)
Comment (clanam) [reply app.py]: nope! i figured it's all /* TODO TODO CODE HERE TODO */

File: app.py (hunk 2 above)
Comment (nifengle) [reply app.py]: I don't want to commit the seed file, though

File: app.py (hunk 3 above)
Comment (gmkohler) [reply app.py]: ok cool

**Comment by clanam** [comment]
> You need to re-build the requirements if you're including flask, etc.:
> 
> ```shell
> pip freeze > requirements.txt
> ```

oh. i thought it just didn't change the requirements. i really don't know python. ^^'

**Comment by gmkohler** [comment]
@clanam the package management is a bit lacking

**Comment by clanam** [comment]
> @clanam the package management is a bit lacking

i think i got it! i probably screwed up the branch history though. 😓

File: requirements.txt [hunk 4]
```diff
@@ -3,9 +3,15 @@ async-timeout==3.0.1
 attrs==19.1.0
 certifi==2019.6.16
 chardet==3.0.4
+Click==7.0
```
Comment (clanam) [inline requirements.txt]: from this:

```
pip install Flask==1.0.2
Requirement already satisfied: Flask==1.0.2 in ./env/lib/python3.7/site-packages (1.0.2)
Collecting Werkzeug>=0.14 (from Flask==1.0.2)
  Using cached https://files.pythonhosted.org/packages/d1/ab/d3bed6b92042622d24decc7aadc8877badf18aeca1571045840ad4956d3f/Werkzeug-0.15.5-py2.py3-none-any.whl
Collecting itsdangerous>=0.24 (from Flask==1.0.2)
  Using cached https://files.pythonhosted.org/packages/76/ae/44b03b253d6fade317f32c24d100b3b35c2239807046a4c953c7b89fa49e/itsdangerous-1.1.0-py2.py3-none-any.whl
Collecting click>=5.1 (from Flask==1.0.2)
  Using cached https://files.pythonhosted.org/packages/fa/37/45185cb5abbc30d7257104c434fe0b07e5a195a6847506c074527aa599ec/Click-7.0-py2.py3-none-any.whl
Collecting Jinja2>=2.10 (from Flask==1.0.2)
  Using cached https://files.pythonhosted.org/packages/1d/e7/fd8b501e7a6dfe492a433deb7b9d833d39ca74916fa8bc63dd1a4947a671/Jinja2-2.10.1-py2.py3-none-any.whl
Collecting MarkupSafe>=0.23 (from Jinja2>=2.10->Flask==1.0.2)
  Using cached https://files.pythonhosted.org/packages/ce/c6/f000f1af136ef74e4a95e33785921c73595c5390403f102e9b231b065b7a/MarkupSafe-1.1.1-cp37-cp37m-macosx_10_6_intel.whl
Installing collected packages: Werkzeug, itsdangerous, click, MarkupSafe, Jinja2
Successfully installed Jinja2-2.10.1 MarkupSafe-1.1.1 Werkzeug-0.15.5 click-7.0 itsdangerous-1.1.0
```

**Pull request merged**